packages/dx-react-grid/docs/guides/lazy-loading.md

```diff
@@ -0,0 +1,43 @@
+# React Grid - Virtual Scrolling with Remote Data: Lazy Loading
+
+Lazy loading enables the Grid component to load row data in portions, once rows enter the viewport.
+
+## Related Plugins
+
+The following plugins implement lazy loading:
+
+- [VirtualTableState](../reference/virtual-table-state.md) - requests data from the server
+- [VirtualTable](../reference/virtual-table.md) - renders loaded rows
+
+Note that the [plugin order](./plugin-overview.md#plugin-order) is important.
+
+## Basic Setup
+
+This feature works only in [controlled mode](controlled-and-uncontrolled-modes.md).
+
+Grid rows are loaded in pages. The Grid maintains the visible, the previous, and the next page. The number of rows in each page is specified in the `pageSize` property. The less the `pageSize` is, the more frequently the Grid requests rows. The total number of rows in the data set is specified in the `totalRowCount` property.
+
+Lazy loading is initiated by the `getRows` function that accepts the `skip` and `take` parameters. They specify how many rows to skip from the start of the data set and how many rows to load.
+
+The `loading` property should indicate whether there is an active request. Once the request is completed, pass the loaded rows to the `Grid` plugin's `rows` property. In addition, pass the `skip` value to the `VirtualTableState` plugin's `skip` property so that the Grid can understand where to start rendering the rows.
```

Comment:
```suggestion
The `loading` property should indicate whether there is an active request. Once the request is completed, pass the loaded rows to the `Grid` plugin's `rows` property. In addition, pass the `skip` value to the `VirtualTableState` plugin's `skip` property, so that the Grid can understand where to begin rendering rows.
```